imports/ui/components/includes/header.js

```diff
@@ -60,6 +60,7 @@ Template.header.onRendered(function () {
 
         $('.mejs__time-rail').append('<span class="mejs__broadcast">Live ' +
         'Broadcast</span>');
+        // $('.mejs__time-rail').addClass('mejs__time-rail__correction');
```

Comment:
So, if we're doing away with it let's delete altogether instead of commenting out.